spec/features/profile/update_password_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+describe 'password updating for the superadmin' do
+  before :each do
+    @superadmin = FactoryGirl.create(:user, superadmin: true, username: 'admin', password: 'myadminpassword')
+    @old_encrypted_password = @superadmin.encrypted_password
+
+    login_as @superadmin
```

Comment:
Ich setze es um, sobald ich weiß, was daran schöner ist. login_as prüft anscheinend immer noch etwas, bevor es dann login_with ausführt, aber in den View-Tests ist login_with nicht bekannt ... Mal schauen, aber danke für den Input :)